<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataMiningApp</class>
 <widget class="QWidget" name="DataMiningApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Mining Application</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>751</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Mining Application: </string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>711</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>minsup: </string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>save to</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="6">
      <widget class="QProgressBar" name="progressBar">
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignTop">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Communication Information:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_minsup">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_cs_prog">
       <property name="text">
        <string>No CS computations</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Database Path</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="BT_start">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>dom:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Secure Computation variables (M,P):</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="7">
      <widget class="QTextBrowser" name="textBrowser">
       <property name="lineWidth">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_M">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="spinBox_P">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QCheckBox" name="checkBox_server">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>server</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="4">
      <widget class="QLineEdit" name="lineEdit_save_url">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLineEdit" name="lineEdit_IPPort">
       <property name="text">
        <string>IP,Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="spinBox_dom">
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>1100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QLineEdit" name="lineEdit_db_url">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="1" colspan="4">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="spacing">
        <number>8</number>
       </property>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_info">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>mining info</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_results">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Show itemsets</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_results_file">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Open Results File</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButton_Log">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Log</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
